src/models/transfer/facade.js

```diff
@@ -598,6 +598,26 @@ const timeoutExpireReserved = async (segmentId, intervalMin, intervalMax) => {
     const knex = await Db.getKnex()
     await knex.transaction(async (trx) => {
       try {
+        // const logger = Logger.child({
```

Comment:
I would clean up the console.logs.
These Logger.debug statements seemed that they were useful in fixing this, wouldn't they be fine to leave uncommented at the debug level?

Reply:
Will do.